tests/integration/plugins.js

```diff
@@ -416,4 +421,29 @@ describe('Integration: plugins', () => {
       `)
     })
   })
+
+  describe('jss-nested', () => {
```

Comment:
this test actually belongs to jss-nested

Reply:
alternatively we could describe the same scenario using plugins api instead of jss-nested plugin

Reply:
I think that this test should be here because it tests the way how rules apply in some cases, so the same scenario instead of jss-nested looks better for me